creativecloud/features/genfill/genfill-interactive.js

```diff
@@ -71,15 +71,14 @@ export default async function decorateGenfill(el) {
   const interactiveContainer = el.querySelector('.interactive-container');
   const allP = interactiveContainer.querySelectorAll('.media:first-child p');
   const pMetadata = [...allP].filter((p) => !p.querySelector('picture'));
-  const [enticementMode, enticement, timer = null] = [...pMetadata];
-  enticementMode.classList.add('enticement-mode');
```

Comment:
We need to remove the class 'enticement-mode' from the CSS if it is no longer being used